emulator-run-cmd/src/sdk.ts

```diff
@@ -76,16 +76,7 @@ export abstract class BaseAndroidSdk implements AndroidSDK {
     }
 
     async acceptLicense(): Promise<any> {
-        await execWithResult(`mkdir -p ${this.androidHome()}/licenses`)
-
-        await writeLicenseFile(`${this.androidHome()}/licenses/android-sdk-license`, "8933bad161af4178b1185d1a37fbf41ea5269c55\n" +
-            "d56f5187479451eabf01fb78af6dfcb131a6481e\n" +
-            "24333f8a63b6825ea9c5514f83c2829b004d1fee")
-        await writeLicenseFile(`${this.androidHome()}/licenses/android-sdk-preview-license`, "84831b9409646a918e30573bab4c9c91346d8abd\n")
-        await writeLicenseFile(`${this.androidHome()}/licenses/intel-android-extra-license`, "d975f751698a77b662f1254ddbeed3901e976f5a\n")
-        await writeLicenseFile(`${this.androidHome()}/licenses/mips-android-sysimage-license`, "e9acab5b5fbb560a72cfaecce8946896ff6aab9d\n")
-        await writeLicenseFile(`${this.androidHome()}/licenses/google-gdk-license`, "33b6a2b64607f11b759f320ef9dff4ae5c47d97a\n")
-        await writeLicenseFile(`${this.androidHome()}/licenses/android-googletv-license`, "601085b94cd77f0b54ff86406957099ebe79c4d6\n")
+        await execWithResult(`bash -c \\\"yes | ${this.androidHome()}/tools/bin/sdkmanager --licenses"`);
```

Comment:
This was already broken before so unfortunately we need to specify licenses explicitly

Reply:
Has been working for me, wonder what the trouble was

Reply:
The "dead" code removed was only the `writeLicenseFile` function which I was able to determine was only being used here

Reply:
If you look on the diff for this PR, you'll see lot's of other code removed